content/2017/12/14/introducing-herman.rst

```diff
@@ -0,0 +1,417 @@
+using an ``@annotation`` syntax.
+Here are just a few of their options:
+
+.. code:: scss
+
+    /// @group buttons
+    /// @access public
+    /// @param {color} $color [green] - The background color for the button
+    /// @example scss
+    ///   .button {
+    ///     @include call-to-action(red);
+    ///   }
+
+Herman supports all the SassDoc annotations,
+which focus on documenting Sass abstractions:
+variables, functions, and mixins.
+From there, we've been adding style guide features of our own.
+
+
+Visualizing design tokens
+-------------------------
+
+A "design token" is an abstract bit of design language
+that normally becomes visible only when applied to
+more explicit patterns and components.
+Colors, fonts, scales, and grid-configurations
+act as "sub-atomic" aspects of a design system --
+often stemming directly from the brand,
+before any UI components have been built.
+This is a good place to start defining your system,
+and Herman can help you visualize these abstractions.
+
+.. image:: /static/images/blog/2017/herman-intro/colors.jpg
+   :alt: Herman color palettes
+   :class: extend-large img-border img-shadow img-spacing
+   :target: /herman/docs/demo_colors.html
+
+.. image:: /static/images/blog/2017/herman-intro/sizes.jpg
+   :alt: Herman size palettes
+   :class: extend-small img-border img-shadow
+   :target: /herman/docs/demo_sizes.html
+
+Herman adds annotations for colors (``@colors``),
+fonts (``@fonts``), sizes (``@sizes``), and ratios (``@ratios``).
+In order to display the data,
+you will need to export all your Sass tokens to json --
+using our provided Sass utilities.
+We'll continue working to make this step
+as smooth and automated as possible,
+but you can find full details in our `Herman documentation`_.
+
+.. _Herman documentation: /herman/docs/
+
+.. code:: scss
+
+  /// @colors brand-primaries
+  /// @font my-font (regular, bold)
+  ///   <any html head required for CDN font imports>
+  /// @sizes my-spacing
+  /// @ratios my-modular-scale
+
+
+SVG icons
+---------
+
+Icons live somewhere between design tokens
+(the icons that are available)
+and atomic patterns
+(how icons are added to the markup).
+At the token level, we provide an ``@icons`` annotation
+to display all the SVG icons in a given folder:
+
+.. code:: scss
+
+  /// @icons path-to/my-assets/svg/
+
+.. image:: /static/images/blog/2017/herman-intro/icons.jpg
+   :alt: Herman icon palettes
+   :class: extend-small img-border img-shadow
+   :target: /herman/docs/demo_icons.html
+
+
+Rendered output & examples
+--------------------------
+
+At the pattern level,
+we provide more robust tools
+for rendering code examples and live patterns.
+While SassDoc only documents Sass abstractions,
+Herman allows
+`documentation of CSS selectors and markup patterns`_.
+
+.. _documentation of CSS selectors and markup patterns: /herman/docs/demo_examples.html
+
+.. image:: /static/images/blog/2017/herman-intro/examples.jpg
```

Comment:
Where is this screenshot taken from? I don't think it's using the latest version of Herman, because the output HTML isn't being pretty-printed.

Reply:
oh, that was probably on Time Designer. I tried to clean up the output in dev tools, but maybe better to create a better example in the Herman docs or elsewhere?

Reply:
MCS and OddSite (and Herman itself) are all on the latest version.

Reply:
yeah, Herman doesn't have any nice simple patterns to show this off (though we could create a demo). OddSite doesn't have any documented yet - and broke when I tried (see #173). MedCurbside examples don't seem to use njk yet…